README.md

```diff
@@ -18,7 +18,11 @@ The Node.js version to install. "0.10" is the default and works on all supported
 
     nodejs_install_npm_user: "{{ ansible_ssh_user }}"
 
-The user for whom the npm packages will be installed can be set here, this defaults to ansible_ssh_user
+The user for whom the npm packages will be installed can be set here, this defaults to ansible_user
```

Comment:
I'm happy to bump to 2.0 at this point; there are few (if any) reasons for people to still be on 1.9.x.